extendr-api/src/robj/into_robj.rs

```diff
@@ -710,6 +710,16 @@ impl From<Vec<Rstr>> for Robj {
     }
 }
 
+#[cfg(feature = "faer")]
+impl From<faer::Mat<f64>> for Robj {
+    /// Convert a faer Mat<f64> into Robj.
+    fn from(mat: faer::Mat<f64>) -> Self {
+        mat.col_chunks(1)
+            .flat_map(|c| c.row_chunks(1).map(|r| r.read(0, 0)))
+            .collect_robj()
```

Comment:
I'm not sure if this is the best/fastest way to iterate, but it works and can always be revisited later.